frontend/src/_components/DynamicForm.jsx

```diff
@@ -138,6 +138,7 @@ const DynamicForm = ({
           theme: darkMode ? 'monokai' : lineNumbers ? 'duotone-light' : 'default',
           placeholder,
           height,
+          componentName: `datasources/${selectedDataSource?.kind}::${$key ?? ''}`,
```

Comment:
I think hardcoding `datasources` here might not be a good idea as we are using this for dataqueries too

Reply:
Yes, makes sense.